frontend/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG NODE_VERSION=17.0.0
+
+FROM node:${NODE_VERSION}-slim as base
```

Comment:
A bit funny the slim can be used to build with. I guess that is node specific. :)

Reply:
Made test locally and total build time of image was 200 s. using only slim and 180 second using non-slim for build.

Will use only slim for simplicity.